sysutils/pfSense-pkg-node_exporter/files/usr/local/share/pfSense-pkg-node_exporter/node_exporter.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
```

Comment:
This file seems to be a copy of sysutils/node_exporter init script, we don't need to carry it here